commit 926b507fa5c3192b1b68fab5910cbd3ba9377c97
Author: Paul Eggert <eggert@cs.ucla.edu>
Date:   Thu Sep 5 14:36:02 2024 -0700

    "Apr", not "April", in IN column
    
    * northamerica (Rule): Use "Apr", not "April", in the IN column.
    Both forms are valid, but "Apr" is more consistent.
    Problem reported by Howard Hinnant.

diff --git a/northamerica b/northamerica
index 01f392e0..1af874b6 100644
--- a/northamerica
+++ b/northamerica
@@ -2631,7 +2631,7 @@ Zone America/Dawson	-9:17:40 -	LMT	1900 Aug 20
 # http://puentelibre.mx/noticia/ciudad_juarez_cambio_horario_noviembre_2022/
 
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
-Rule	Mexico	1931	only	-	April	30	0:00	1:00	D
+Rule	Mexico	1931	only	-	Apr	30	0:00	1:00	D
 Rule	Mexico	1931	only	-	Oct	1	0:00	0	S
 Rule	Mexico	1939	only	-	Feb	5	0:00	1:00	D
 Rule	Mexico	1939	only	-	Jun	25	0:00	0	S
